x-pack/plugins/apm/dev_docs/overflow_bucket_setup.md

```diff
@@ -1,11 +1,112 @@
+# Table of contents
+1. [Summary](#summary)
+2. [Pre-requisites](#prerequisites)
+3. [Using tilt](#tilt)
+4. [Other ways](#other)
+
 ## Summary
 
 8.7 introduced Overflow Buckets for Metrics. The below setup would enable generating the overflow buckets for the metrics indices.
 
-### Pre-requisites
+## Pre-requisites
 
 - Install golang and set go path properly
```

Comment:
There are now two pre-requisite sections. Could this be merged with the other section?